src/Geopilot.Frontend/src/pages/admin/mandateDetail.tsx

```diff
@@ -0,0 +1,252 @@
+import { FieldEvaluationType, Mandate, Organisation, ValidationSettings } from "../../api/apiInterfaces.ts";
+import { useApi } from "../../api";
+import { useGeopilotAuth } from "../../auth";
+import { FormAutocompleteValue } from "../../components/form/formAutocomplete.tsx";
+
+export const MandateDetail = () => {
+  const { t } = useTranslation();
+  const { user } = useGeopilotAuth();
+  const formMethods = useForm({ mode: "all" });
+  const { showPrompt } = useContext(PromptContext);
+  const { fetchApi } = useApi();
+  const navigate = useNavigate();
+  const { id } = useParams<{
+    id: string;
+  }>();
+
+  const [mandate, setMandate] = useState<Mandate>();
+  const [organisations, setOrganisations] = useState<Organisation[]>();
+  const [fileExtensions, setFileExtensions] = useState<string[]>();
+
+  const loadMandate = useCallback(() => {
+    if (id !== "0") {
+      fetchApi<Mandate>(`/api/v1/mandate/${id}`, { errorMessageLabel: "mandateLoadingError" }).then(setMandate);
+    } else {
+      setMandate({ id: 0, name: "", organisations: [], fileTypes: [], coordinates: [], deliveries: [] });
+    }
+  }, [fetchApi, id]);
+
+  const loadOrganisations = useCallback(() => {
+    fetchApi<Organisation[]>("/api/v1/organisation", { errorMessageLabel: "organisationsLoadingError" }).then(
+      setOrganisations,
+    );
+  }, [fetchApi]);
+
+  const loadFileExtensions = useCallback(() => {
+    fetchApi<ValidationSettings>("/api/v1/validation", { errorMessageLabel: "fileTypesLoadingError" }).then(
+      validation => {
+        setFileExtensions(validation?.allowedFileExtensions);
+      },
+    );
+  }, [fetchApi]);
+
+  useEffect(() => {
+    if (user?.isAdmin) {
+      if (mandate === undefined) {
+        loadMandate();
+      }
+      if (organisations === undefined) {
+        loadOrganisations();
+      }
+      if (fileExtensions === undefined) {
+        loadFileExtensions();
+      }
+    }
+  }, [fileExtensions, loadFileExtensions, loadMandate, loadOrganisations, mandate, organisations, user?.isAdmin]);
+
+  async function saveMandate(data: FieldValues, closeAfterSave = false) {
+    if (id !== undefined) {
+      const mandate = data as Mandate;
+      mandate.deliveries = [];
+      mandate.organisations = data["organisations"]?.map(
+        (value: FormAutocompleteValue) => ({ id: value.key }) as Organisation,
+      );
+      mandate.fileTypes = data["fileTypes"]?.map((value: FormAutocompleteValue) => value.name as string);
+      mandate.id = parseInt(id);
+      await fetchApi("/api/v1/mandate", {
+        method: mandate.id === 0 ? "POST" : "PUT",
+        body: JSON.stringify(mandate),
+        errorMessageLabel: "mandateSaveError",
+      }).then(response => {
+        const mandateResponse = response as Mandate;
+        if (!closeAfterSave) {
+          if (id === "0") {
+            navigate(`/admin/mandates/${mandateResponse.id}`);
+          } else {
+            loadMandate();
+          }
+        }
+      });
```

Comment:
PP: await und then nicht mixen
```suggestion
      const response = await fetchApi("/api/v1/mandate", {
        method: mandate.id === 0 ? "POST" : "PUT",
        body: JSON.stringify(mandate),
        errorMessageLabel: "mandateSaveError",
       });

       const mandateResponse = response as Mandate;
       if (!closeAfterSave) {
          if (id === "0") {
            navigate(`/admin/mandates/${mandateResponse.id}`);
          } else {
            setMandate(mandateResponse);
          }
        }
      };
```

Reply:
Würde ich gerne mit #361 anschauen, weil es sonst ein zu grosses Durcheinander gibt.

Reply:
https://github.com/GeoWerkstatt/geopilot/pull/361/commits/10a47217781e00ad0bea75abe40857c2e2720d48